cmake/Modules/PothosUtil.cmake

```diff
@@ -97,22 +97,24 @@ function(POTHOS_MODULE_UTIL)
         endif(GIT_FOUND)
     endif()
 
+    #setup module build and install rules
+    add_library(${POTHOS_MODULE_UTIL_TARGET} MODULE ${POTHOS_MODULE_UTIL_SOURCES})
+    target_link_libraries(${POTHOS_MODULE_UTIL_TARGET} PRIVATE Pothos ${POTHOS_MODULE_UTIL_LIBRARIES})
+    set_target_properties(${POTHOS_MODULE_UTIL_TARGET} PROPERTIES DEBUG_POSTFIX "") #same name in debug mode
+
     #version specified, build into source file
     if (POTHOS_MODULE_UTIL_VERSION)
         message(STATUS "Module ${POTHOS_MODULE_UTIL_TARGET} configured with version: ${POTHOS_MODULE_UTIL_VERSION}")
         set(version_file "${CMAKE_CURRENT_BINARY_DIR}/Version.cpp")
         file(WRITE "${version_file}" "#include <Pothos/Plugin/Module.hpp>
             static const Pothos::ModuleVersion register${MODULE_TARGET}Version(\"${POTHOS_MODULE_UTIL_VERSION}\");
         ")
-        list(APPEND POTHOS_MODULE_UTIL_SOURCES "${version_file}")
+        target_sources(${POTHOS_MODULE_UTIL_TARGET} PRIVATE "${version_file}")
     endif()
 
     #always enable docs if user specifies doc sources
     if (POTHOS_MODULE_UTIL_DOC_SOURCES)
         set(POTHOS_MODULE_UTIL_ENABLE_DOCS TRUE)
-    #otherwise doc sources come from the regular sources
-    else()
-        set(POTHOS_MODULE_UTIL_DOC_SOURCES ${POTHOS_MODULE_UTIL_SOURCES})
```

Comment:
This breaks PothosFlow support for the majority of current modules, as they don't provide their own custom doc sources.

Reply:
that was not intentional, good catch